backend/core/views.py

```diff
@@ -196,16 +196,20 @@ def image_download_api(request):
     Args:
         dataset_id: int - id of the dataset
         source : str - source url of OAM if present or any other URL - Optional
+        zoom_level : int - zoom level default is 19
     Returns:
         Download status
     """
     print(request.data)
 
     serializer = ImageDownloadSerializer(data=request.data)
     if serializer.is_valid(raise_exception=True):
-        dataset_id = request.data.get("dataset_id")
+        dataset_id = int(request.data.get("dataset_id"))
         # get source imagery url if supplied else use maxar
         source = request.data.get("source", "maxar")
+        zoom_level = int(request.data.get("zoom_level", 19))
```

Comment:
so the zoom is a parameter now and default value is 19

Reply:
Yes true !